src/tests/JIT/Performance/JittedMethodsCountingTest/JittedMethodsCountingTest.cs

```diff
@@ -34,6 +50,12 @@ public static int TestEntryPoint()
         return jits > 0 && jits <= MAX_JITTED_METHODS_ACCEPTED ? 100 : 101;
     }
 
+    private static bool IsReadyToRunEnvSet()
+    {
+        string? dotnetR2R = Environment.GetEnvironmentVariable("DOTNET_ReadyToRun");
+        return (dotnetR2R is null || dotnetR2R == "1") ? true : false;
```

Comment:
I have a vague feeling (you probably know that better than I do) that on Linux you can end up with either ```null``` or ```empty string``` both of which should probably have the same behavior i.o.w. we should probably use ```string.IsNullOrEmpty``` (or return to my original lame suggestion that inequality to ```"0"``` is good enough).

Reply:
It also returns `null` on Linux. However, since any kind of empty is equivalent to "1" in this case, we can play it safer by going the `string.IsNullOrEmpty()` route instead :+1: